src/test/groovy/ca.cutterslade.gradle.analyze/AnalyzeDependenciesPluginSpec.groovy

```diff
@@ -99,9 +99,34 @@ class AnalyzeDependenciesPluginSpec extends Specification {
 
         where:
         configuration    | expectedResult
-        "compile"        | "unusedDeclaredArtifacts"
-        "implementation" | "unusedDeclaredArtifacts"
-        "compileOnly"    | "unusedDeclaredArtifacts"
+        "compile"        | "Unused declared"
+        "implementation" | "Unused declared"
+        "compileOnly"    | "Unused declared"
+        "runtimeOnly"    | SUCCESS
+    }
+
+    @Unroll
+    def "unused main dependency with permitUnusedDeclared declared with #configuration results in #expectedResult"(String configuration, String expectedResult) {
```

Comment:
Worth commenting this seeming "garbage" here for non-Groovy-saavy reviewers like me?

Reply:
lol, it's just like every other test in this file. Should be fine.